nextcloudappstore/core/static/assets/css/fonts.css

```diff
@@ -0,0 +1,120 @@
+/* Generated by Font Squirrel (https://www.fontsquirrel.com) on July 12, 2016 */
+
+
+
+@font-face {
+    font-family: 'open_sanssemibold_italic';
```

Comment:
We never ever use italic faces anywhere. :) Only Light and Semibold. Emphasis (or de-emphasis) can be achieved by opacity / greyscale, which looks a lot better.